src/components/search/search.css

```diff
@@ -0,0 +1,67 @@
+.search {
+  padding: 8px 12px;
+  display: flex;
+  align-items: center;
+  gap: 12px;
+
+  border-radius: 8px;
+  border: 1px solid;
+
+  transition: 
+    border-color .2s ease-in-out;
+}
+
+.search.light {
+  border-color: var(--outline-base);
+  background-color: var(--base-white);
+  color: var(--base-black);
+}
+
+.search.light:focus-within {
+  border-color: var(--blue-base) !important;
+}
+
+.search.dark {
+  border-color: var(--outline-inverse);
+  background-color: var(--base-gray);
+  color: var(--base-white);
+}
+
+.search.dark:focus-within {
+  border-color: var(--blue-inverse) !important;
+}
+
+.search input {
+  all: unset;
+  width: 100%;
+  font-family: "Karla", sans-serif;
```

Comment:
Não sei exatamente se as fontes são diferentes das padrões, mas seria legal colocar as fontes que usamos num arquivo separado sipa

Não precisa ser agr, mas deixar mapeado sipa no GitHub Issues já seria luz

Reply:
É, eu acho que tem algumas coisas que da pra extrair, se a gente for fazer isso, strings tbm seria bom